neurom/stats.py

```diff
@@ -48,3 +49,28 @@ def fit(data, distribution='norm'):
     '''
     params = getattr(_st, distribution).fit(data)
     return params, _st.kstest(data, distribution, params)
+
+
+def optimal_distribution(data):
+    '''Calculate the parameters of a fit of different distributions to a data set
+       and returns the distribution of the minimal ks-distance.
+
+    Parameters:
+        data: array of data points to be fitted
+
+    Returns:
+        tuple (optimal, parameters) of data to fitted distribution
+
+    Note:
+        Uses Kolmogorov-Smirnov test to estimate distance and p-value.
+
+        The distributions that are tested are uniform, normal and exponential.
+    '''
+
+    distr_to_check = ['norm', 'expon', 'uniform']
+
+    fit_d = {d: fit(data, d) for d in distr_to_check}
+
+    optimal = fit_d.keys()[_np.argmin([error[1][0] for error in fit_d.values()])]
+
+    return optimal, fit_d[optimal][0]
```

Comment:
Yes, I agree, but this is a different issue :) If you want to use one of the predefined distributions you need to have a measure of it. So I'll just return the full fit data here.